### PR TITLE
aspcud: update to 1.9.5

### PR DESCRIPTION
**Pull request by percontation**
Uses new homepage and github

Aspcud depends on the port gringo, which is now provided as part of
a larger "clingo" project, so the gringo port has been updated by
replacing it with a new clingo port.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 11.5.1 20G80 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/aspcud/Portfile [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
     Aspcud is a solver for package dependencies. A package universe and a request \
     to install, remove, or upgrade packages have to be encoded in the CUDF format. \
     Such a CUDF document can then be passed to aspcud along with an optimization \
     criteria to obtain a solution to the given package problem.
 
-homepage            http://potassco.sourceforge.net
-master_sites        sourceforge:project/potassco/aspcud/${version}/
+homepage            https://potassco.org/aspcud/
+github.tarball_from archive
+worksrcdir          ${name}-${version}
```
Comment (reneeotten) [inline devel/aspcud/Portfile:22]: is this line necessary - I think the github PG should set this correctly

File: devel/aspcud/Portfile [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
 
-depends_build-append \
-                    port:re2c
-depends_lib         port:clasp \
-                    port:gringo
+compiler.cxx_standard   2014
 
-cmake.out_of_source yes
+depends_build-append port:re2c
+depends_lib          port:clingo
```
Comment (reneeotten) [inline devel/aspcud/Portfile:34]: this line and the one above do not seem to conform to the modeline (i.e., multiples of four spaces indentation)

File: math/clingo/Portfile [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+description         An Answer Set Programming system to ground and solve logic programs.
+long_description \
+    Clingo is part of the Potassco project for Answer Set Programming (ASP). \
+    ASP offers a simple and powerful modeling language to describe combinatorial \
+    problems as logic programs. The clingo system then takes such a logic program \
+    and computes answer sets representing solutions to the given problem.
+
+homepage            https://potassco.org/clingo/
+github.tarball_from archive
+worksrcdir          ${name}-${version}
```
Comment (reneeotten) [inline math/clingo/Portfile:21]: as above, this shouldn't be needed

File: devel/aspcud/Portfile (hunk 1 above)
Comment (percontation) [reply devel/aspcud/Portfile]: thx. Last time I wrote a portfile was before the github PG existed 😛

**Review by reneeotten — APPROVED** [review]

**Comment by reneeotten** [comment]
thanks @percontation !

**Pull request merged**